pkg/controller/import-controller.go

```diff
@@ -699,6 +723,42 @@ func getEndpoint(pvc *corev1.PersistentVolumeClaim) (string, error) {
 	return ep, nil
 }
 
+// returns the OpenShift cluster wide proxy object
+func getClusterWideProxy(log logr.Logger, client client.Client) (*ocpconfigv1.Proxy, error) {
+	// Get the Openshift cluster-wide Proxy obj.
+	u := &unstructured.Unstructured{}
+	u.SetGroupVersionKind(schema.GroupVersionKind{
+		Group:   ClusterWideProxyAPIGroup,
+		Kind:    ClusterWideProxyAPIKind,
+		Version: ClusterWideProxyAPIVersion,
+	})
+	if err := client.Get(context.TODO(), types.NamespacedName{Name: ClusterWideProxyName}, u); err != nil {
```

Comment:
I do not like that an import-controller has to to know it is running on OpenShift and has to depend on its api. But I do not want to suggest any overengineered solution for a simple task of gathering variables.

Is it possible to make a set of starndard CDI proxy parameters and inject these variables from outside? If not then maybe we can at least hide the `getClusterWideProxy` in some kind of utility and make it return a very simple struct with three fields: `ImporterHTTPSProxy`, `ImporterHTTPProxy` and `ImporterNoProxy`.

Reply:
It makes sense and I moved that to `controller/util.go`